packages/app_ui/lib/src/dialogs/bottom_animated_dialog.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class BottomAnimatedDialog {
+  static void show(BuildContext context, {required Widget child}) {
+    showGeneralDialog(
+      barrierLabel: 'BottomAnimatedDialog',
+      barrierDismissible: true,
+      barrierColor: Colors.black.withOpacity(0.5),
+      transitionDuration: const Duration(milliseconds: 500),
+      context: context,
+      pageBuilder: (context, anim1, anim2) {
+        return Align(
+          alignment: Alignment.bottomCenter,
+          child: Container(
+            height: 300,
```

Comment:
hardcoded dialog height may be tricky when you put some bigger child there. Let's leave it for now and maybe you will refactor that in the future and need will come.